### ISSUE TITLE
[v2] Problem with latest momentjs release

### ISSUE THREAD

**Issue by tcastelli** [gadicc/meteor-messageformat#189]
I'm getting this while trying to change locale with the latest momentjs library 2.11.0

`````javascript
jag_pince.js:147 debug: [msgfmt] fetchLocale arrived 421ms after request
jag_pince.js:147 debug: [msgfmt] locale set to es_ES

Uncaught ReferenceError: module is not definedloadLocale @ moment.js:273
locale_locales__getLocale @ moment.js:339
locale_locales__getSetGlobalLocale @ moment.js:293
(anonymous function) @ messageformat-client.js:455
(anonymous function) @ raix_eventemitter.js:104
_.each._.forEach @ underscore.js:105
_runCallbacks @ raix_eventemitter.js:100
EventEmitter.emit @ raix_eventemitter.js:129
localeReady @ messageformat-client.js:244
msgfmt.fromServer @ messageformat-client.js:385
jag_pince.js:147 debug: [msgfmt] Meteor.startup(), 1062ms after script loading
````

**Comment by DSpeichert** [comment]
Hi @tcastelli,
Thanks for submitting the issue.
Do you load es_ES language for momentjs? Momentjs does not come with languages by default - so you might or might not have them, depending on what package you use.

**Comment by tcastelli** [comment]
No,  i havent added any aditional language to momentjs,  but with previous
official package the error doesnt happen (and it also doesnt have any
locales included)

**Comment by gadicc** [comment]
It's a momentjs bug, already fixed in the patch release momentjs 2.11.1 - see https://github.com/moment/moment/issues/2862.
